README.md

```diff
@@ -28,21 +32,37 @@ Also check the [config.yaml documentation](https://github.com/haskell/ghcup-hs/b
 
 ## For contributors
 
-### Adding a new GHC version
+Most bindists are built downstream by the GHCup developers, e.g.:
+
+* https://github.com/stable-haskell/haskell-language-server
+* https://github.com/stable-haskell/cabal
+* https://github.com/stable-haskell/stack
+
+For GHC bindists there is no automation yet and it is done manually (e.g. for FreeBSD and Alpine i386).
+
+That makes contributions harder, because your pull requests to the metadata will be partial.
+
+## For upstream developers
+
+GHC, Cabal, HLS and stack developers distribute their own bindists via the `ghcup-vanilla-A.B.C.yaml` files. These
```

Comment:
Does this section only covers PRs to the vanilla channel. In that case, I'd suggest to mention it in the title:
```suggestion
## For upstream developers: the `vanilla` channel

GHC, Cabal, HLS and stack developers distribute their own bindists via the `ghcup-vanilla-A.B.C.yaml` files. These
```

After this fix it may be good to change the previous title too, e.g.:
```
## For contributors: the main channel
```

The specifics of this may be improved but the general idea is putting more emphasis on the channels.

Reply:
Everyone can provide PRs for whatever they want. I'm just outlining who is responsible for maintaining which metadata.

Reply:
> Everyone can provide PRs for whatever they want.

If those PRs will not be accepted because the author doesn't have a permission to run a pipeline, this makes no sense. "Can provide PRs" is not the same as "can contribute" (as in: influence the contents of the repo), and I thought you are describing the latter in this README.